### PR TITLE
[Technical] Normalise line endings in repository

### PR DESCRIPTION
**Pull request by mjcheetham**
The `.gitattributes` file is the one created by VS.
Setting all text files to have 'auto' line ending normalisation (and UTF-8 encoding).
This is ensure that the repository will contain `LF` line endings, but will be transformed to the native endings on checkout (i.e., `CRLF` on Windows and `LF` on *NIX).
Also normalised those files which has become inconsistent.

There are no changes (other than whitespace/eol) in this pull request.

See [here](https://help.github.com/articles/dealing-with-line-endings/) for more information.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+#
+# Merging from the command prompt will add diff markers to the files if there
+# are conflicts (Merging from VS is not affected by the settings below, in VS
+# the diff markers are never inserted). Diff markers may cause the following 
+# file extensions to fail to load in VS. An alternative would be to treat
+# these files as binary and thus will always conflict and require user
+# intervention with every merge. To do so, just uncomment the entries below
+###############################################################################
+#*.sln       merge=binary
+#*.csproj    merge=binary
```
Comment (eyalantzMS) [inline .gitattributes]: Yes the project is not loaded in VS i but you can still edit it (and the fact it's not loaded forces you to address the issue).

**Comment by eyalantzMS** [comment]
Other than the issue with merging projects as binaries LGTM

**Comment by mjcheetham** [comment]
@eyalantzMS The `merge=binary` are all commented out, so this has no effect. This is simply the standard `.gitattributes` file as created by VS for an empty solution under git source control.

File: .gitattributes (hunk 1 above)
Comment (eyalantzMS) [inline .gitattributes]: ...missed the fact that they are commented out.

**Pull request merged**